backend/Squirrel.Core/Squirrel.Core.BLL/Services/ProjectService.cs

```diff
@@ -0,0 +1,90 @@
+﻿using AutoMapper;
+using Squirrel.Core.Common.DTO.Projects;
+using Squirrel.Core.BLL.Interfaces;
+using Squirrel.Core.BLL.Services.Abstract;
+using Squirrel.Core.DAL.Context;
+
+namespace Squirrel.Core.BLL.Services
+{
+    public class ProjectService : BaseService, IProjectService
+    {
+        private readonly SquirrelCoreContext _context;
+        private readonly IMapper _mapper;
+
+        public ProjectService(SquirrelCoreContext context, IMapper mapper): base(context, mapper)
+        {
+            _context = context;
+            _mapper = mapper;
+        }
+
+        public async Task<ProjectDTO> AddProject(ProjectDTO projectDto)
+        {
+            // var projectEntity = _mapper.Map<Project>(projectDto);
+            // projectEntity.Engine = (int)projectDto.Engine;
+            // _context.Projects.Add(projectEntity);
```

Comment:
Create migration with simple projects, please

Reply:
And after migration uncomment all that code

Reply:
Fixed